lib/countly_flutter.dart

```diff
@@ -1267,9 +1294,14 @@ class Countly {
   }
 
   /// Enable campaign attribution reporting to Countly.
-  /// For iOS use 'recordAttributionID' instead of 'enableAttribution'
+  /// Deprecated for iOS, for iOS use 'recordAttributionID' instead.
   /// Should be call before Countly init
   static Future<String?> enableAttribution() async {
+    if (Platform.isIOS) {
+      String error = 'enableAttribution is deprecated, for iOS use recordAttributionID instead';
```

Comment:
This message should not be ios specific.
This change should be moved to the other attribution PR and should not be part of this